profile/profile.yaml

```diff
@@ -0,0 +1,113 @@
+# Preferred name (Characters only)
+name: "Hunain Khalid"
+# Your GitHub repo root URL
+github_repo_root_url: "https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid"
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: "Hello, I am Hunain Khalid a software developer with a passion for machine learning (ML) and artifical intelligence (AI). 
```

Comment:
I would rewrite this intro. I will message you some sample profiles.
-No need to introduce yourself by name, your name is on top of the page.
-Include more details about your technical skills and experience
-Ensure that grammar is correct: consider running your profile through grammarly's checking service